Refactor toto to in-toto - Fix #17

lukpueh opened the pull request:
Finally refactors toto into **in-toto** everywhere. Fixes https://github.com/in-toto/in-toto/issues/17

- Repo name: in-toto
- Package setup.py name: in-toto (pip replaces the hyphen with an underscore)
- Package directory name: in_toto (`import`able as opposed to packages/modules with hyphen)
- CLI interfaces (`import`able)
  - in_toto_run.py
  - in_toto_record.py
  - in_toto_verify.py
with entry points (console scripts) specified in `setup.py`
  - in-toto-run
  - in-toto-record
  - in-toto-verify

lukpueh committed:
Refactor toto to in-toto

- Repo name in-toto
- Package name in_toto (importable)
- CLI interfaces (importable)
  - in_toto_run.py
  - in_toto_record.py
  - in_toto_verify.py
with entry points (console scripts):
  - in-toto-run
  - in-toto-record
  - in-toto-verify

SantiagoTorres commented on setup.py:
```diff
@@ -5,17 +5,24 @@
-           'toto/toto-verify.py']
+  scripts=["in_toto/in_toto_run.py",
+           "in_toto/in_toto_record.py",
+           "in_toto/in_toto_verify.py"],
```
do we still need this section if the entry_points are defined?

SantiagoTorres commented on tox.ini:
```diff
@@ -17,5 +17,5 @@ deps =
     # Integrating tox with setup.py test is as of Oct 2016 discouraged
     # http://tox.readthedocs.io/en/latest/example/basic.html#integration-with-setup-py-test-command
-    # coverage run --source=toto setup.py test
+    # coverage run --source=in-toto setup.py test
```
do we want/need this line?

SantiagoTorres reviewed (changes requested):
Two minor nits, but overall is what we need 👍

JustinCappos commented:
Glad to see it getting addressed!

On Fri, Dec 2, 2016 at 6:04 PM, lukpueh <notifications@github.com> wrote:

> Finally refactors toto into *in-toto* everywhere. Fixes #17
> <https://github.com/in-toto/in-toto/issues/17>
>
>    - Repo name: in-toto
>    - Package setup.py name: in-toto (pip replaces the hyphen with an
>    underscore)
>    - Package directory name: in_toto (importable as opposed to
>    packages/modules with hyphen)
>    - CLI interfaces (importable)
>       - in_toto_run.py
>       - in_toto_record.py
>       - in_toto_verify.py
>       with entry points (console scripts) specified in setup.py
>       - in-toto-run
>       - in-toto-record
>       - in-toto-verify
>
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/in-toto/in-toto/pull/55
> Commit Summary
>
>    - Refactor toto to in-toto
>
> File Changes
>
>    - *M* .gitmodules
>    <https://github.com/in-toto/in-toto/pull/55/files#diff-0> (6)
>    - *M* README.md
>    <https://github.com/in-toto/in-toto/pull/55/files#diff-1> (4)
>    - *M* demo/README.md
>    <https://github.com/in-toto/in-toto/pull/55/files#diff-2> (12)
>    - *M* demo/owner_alice/create_layout.py
>    <https://github.com/in-toto/in-toto/pull/55/files#diff-3> (4)
>    - *M* demo/run_demo.py
>    <https://github.com/in-toto/in-toto/pull/55/files#diff-4> (10)
>    - *R* in_toto/__init__.py
>    <https://github.com/in-toto/in-toto/pull/55/files#diff-5> (0)
>    - *R* in_toto/exceptions.py
>    <https://github.com/in-toto/in-toto/pull/55/files#diff-6> (2)
>    - *R* in_toto/in_toto_record.py
>    <https://github.com/in-toto/in-toto/pull/55/files#diff-7> (16)
>    - *R* in_toto/in_toto_run.py
>    <https://github.com/in-toto/in-toto/pull/55/files#diff-8> (38)
>    - *R* in_toto/in_toto_verify.py
>    <https://github.com/in-toto/in-toto/pull/55/files#diff-9> (49)
>    - *R* in_toto/log.py
>    <https://github.com/in-toto/in-toto/pull/55/files#diff-10> (0)
>    - *R* in_toto/matchrule_validators.py
>    <https://github.com/in-toto/in-toto/pull/55/files#diff-11> (12)
>    - *R* in_toto/models/__init__.py
>    <https://github.com/in-toto/in-toto/pull/55/files#diff-12> (0)
>    - *R* in_toto/models/common.py
>    <https://github.com/in-toto/in-toto/pull/55/files#diff-13> (4)
>    - *R* in_toto/models/layout.py
>    <https://github.com/in-toto/in-toto/pull/55/files#diff-14> (20)
>    - *R* in_toto/models/link.py
>    <https://github.com/in-toto/in-toto/pull/55/files#diff-15> (4)
>    - *R* in_toto/runlib.py
>    <https://github.com/in-toto/in-toto/pull/55/files#diff-16> (18)
>    - *R* in_toto/settings.py
>    <https://github.com/in-toto/in-toto/pull/55/files#diff-17> (2)
>    - *R* in_toto/ssl_commons/__init__.py
>    <https://github.com/in-toto/in-toto/pull/55/files#diff-18> (0)
>    - *R* in_toto/ssl_commons/exceptions.py
>    <https://github.com/in-toto/in-toto/pull/55/files#diff-19> (0)
>    - *R* in_toto/ssl_commons/schema.py
>    <https://github.com/in-toto/in-toto/pull/55/files#diff-20> (0)
>    - *R* in_toto/ssl_commons/tests/test_schema.py
>    <https://github.com/in-toto/in-toto/pull/55/files#diff-21> (0)
>    - *R* in_toto/ssl_crypto
>    <https://github.com/in-toto/in-toto/pull/55/files#diff-22> (0)
>    - *R* in_toto/util.py
>    <https://github.com/in-toto/in-toto/pull/55/files#diff-23> (32)
>    - *R* in_toto/verifylib.py
>    <https://github.com/in-toto/in-toto/pull/55/files#diff-24> (22)
>    - *M* setup.py
>    <https://github.com/in-toto/in-toto/pull/55/files#diff-25> (17)
>    - *M* test/models/test_inspection.py
>    <https://github.com/in-toto/in-toto/pull/55/files#diff-26> (6)
>    - *M* test/models/test_layout.py
>    <https://github.com/in-toto/in-toto/pull/55/files#diff-27> (10)
>    - *M* test/models/test_matchrule.py
>    <https://github.com/in-toto/in-toto/pull/55/files#diff-28> (4)
>    - *M* test/models/test_step.py
>    <https://github.com/in-toto/in-toto/pull/55/files#diff-29> (10)
>    - *M* test/test_in_toto_record.py
>    <https://github.com/in-toto/in-toto/pull/55/files#diff-30> (8)
>    - *M* test/test_runlib.py
>    <https://github.com/in-toto/in-toto/pull/55/files#diff-31> (8)
>    - *M* test/test_verifylib.py
>    <https://github.com/in-toto/in-toto/pull/55/files#diff-32> (12)
>    - *M* tox.ini
>    <https://github.com/in-toto/in-toto/pull/55/files#diff-33> (4)
>
> Patch Links:
>
>    - https://github.com/in-toto/in-toto/pull/55.patch
>    - https://github.com/in-toto/in-toto/pull/55.diff
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/in-toto/in-toto/pull/55>, or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AA0XD51EaCLoOR-_GKuCb_0z5ZQ-QPB-ks5rEKQIgaJpZM4LDFVE>
> .
>

Commit:
Removes scripts from setup.py

We use entry_points which alias the scripts.

Commit:
Removes commented line from tox.ini

We used to run tox with setup.py which is discouraged. The
related command was draged along as a comment, but we can
safely git rid of it.

SantiagoTorres commented:
thanks @lukpueh!
